### PR TITLE
Remove SENDCOM

### LINKED ISSUE #1501: Remove SENDCOM flags
### What new functionality do you need?
Workflow is inconsistent in its use of `SENDCOM` to gate movement of files to COM. We've gotten confirmation from NCO that these gates are no longer needed.

### What are the requirements for the new functionality?
All of the copies to COM that are currently gated by `$SENDCOM` should be changed to just always happen.

### Acceptance Criteria
No more instances of `$SENDCOM` in the workflow

### PR DESCRIPTION
**Pull request by HenryRWinterbottom**
# Description

This PR addresses issue #1501. All instances of `SENDCOM` gates have been removed. The workflow was inconsistent in its use of `SENDCOM` to gate movement of files to `COM` and have been removed based on NCO guidelines.

Resolves #1501.

# Type of change
- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
Will let CI test these changes.

# Checklist
- [ ] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
LGTM.

File: scripts/exgfs_atmos_post.sh [hunk 1]
```diff
@@ -314,11 +311,10 @@ else   ## not_anl if_stime
             fi
           fi
```
Comment (WalterKolczynski-NOAA) [inline scripts/exgfs_atmos_post.sh:304]: These lines now have a smaller indentation than the outer blocks.

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]
Just a small indentation thing.

File: scripts/exgfs_atmos_post.sh (hunk 1 above)
Comment (HenryRWinterbottom) [reply scripts/exgfs_atmos_post.sh:304]: Fixed.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**